h/config.py

```diff
@@ -23,7 +23,7 @@
 )
 
 
-def configure(environ=None, settings=None):
+def configure(environ=None, settings=None):  # pylint: disable=too-many-statements
```

Comment:
For this and `routes` I just added the inline disabling. I don't think they are too bad as they are, just a tad long.